### PR TITLE
add AQI description

### PR DESCRIPTION
**Pull request by killerninja8**
hey team, PFA the aqi description. I've **tried** to explain what it is, how it is derived, why we should care about it, and how this feature/dashboard uses AQI to benefit its user base. 

Also updated one sentence about who the app is intended for (we claim in our user persona/motivation that it is specific for health professionals)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,14 @@ Welcome to our repository! Feel free to look around. If you want to go straight
...
 
 ### Who is it for?
-_Pollution Tracker_ is designed for anyone interested in the environment, public health, or public policy. We hope it enables these individuals to get a clear idea of air quality metrics and their trends. Our project is built on data sourced from OpenDataSoft, guaranteeing both reliability and  transparency.
+_Pollution Tracker_ is intended for use by public health officials but designed for anyone interested in the environment, public health, or public policy. We hope it enables these individuals to get a clear idea of air quality metrics and their trends. Our project is built on data sourced from OpenDataSoft, guaranteeing both reliability and  transparency.
+
+## What is AQI? 
+Air Quality Index (AQI) is an indicator of air quality developed by government health authorities to communicate the level of air pollution in an area. When AQI levels are high, several subgroups are affected by it's negative impacts - children, seniors, those with respiratory and cardiorespiratory illnesses. In times of high AQI, health bodies suggest people stay indoors, use high-efficiency particulate absorbing (HEPA) filters, wear NIOSH N95, and avoid any risk to their health.
```
Comment (meretelutz) [inline README.md:16]:
```suggestion
Air Quality Index (AQI) is an indicator of air quality developed by government health authorities to communicate the level of air pollution in an area. When AQI levels are high, several subgroups are affected by it's negative impacts - children, seniors, and those with respiratory and cardiorespiratory illnesses. In times of high AQI, health bodies suggest people stay indoors, use high-efficiency particulate absorbing (HEPA) filters, wear NIOSH N95, and avoid any risk to their health. 
```

File: README.md [hunk 2]
```diff
@@ -10,7 +10,14 @@ Welcome to our repository! Feel free to look around. If you want to go straight
...
+
+## What is AQI? 
+Air Quality Index (AQI) is an indicator of air quality developed by government health authorities to communicate the level of air pollution in an area. When AQI levels are high, several subgroups are affected by it's negative impacts - children, seniors, those with respiratory and cardiorespiratory illnesses. In times of high AQI, health bodies suggest people stay indoors, use high-efficiency particulate absorbing (HEPA) filters, wear NIOSH N95, and avoid any risk to their health. 
+
+AQI is calculated as the maximum of the index of pollutant $p$, where this index is defined as: $I_p = \left\lfloor \frac{(I_{\text{high}} - I_{\text{low}}) \times (C_p - C_{\text{low}})}{(C_{\text{high}} - C_{\text{low}})} + I_{\text{low}} \right\rfloor \quad$ $\forall p \in [\{\text{CO}_2, \text{CO}, \text{O}_3, \text{PM2.5}, \text{PM10}, \text{SO}_x, \text{NO}_x\}]$. This index is the linear interpolation of concentration between concentration breakpoints ${C}$ for pollutant $p$. However, the values are often represented as one of five groups (labels vary by country): very poor, poor, moderate, good, very good.
```
Comment (meretelutz) [inline README.md:18]:
```suggestion
AQI is calculated as the maximum of the index of pollutant $p$, where this index is defined as: $I_p = \left\lfloor \frac{(I_{\text{high}} - I_{\text{low}}) \times (C_p - C_{\text{low}})}{(C_{\text{high}} - C_{\text{low}})} + I_{\text{low}} \right\rfloor \quad$ $\forall p \in [\{\text{CO}_2, \text{CO}, \text{O}_3, \text{PM2.5}, \text{PM10}, \text{SO}_x, \text{NO}_x\}]$. This index is the linear interpolation of concentration between concentration breakpoints ${C}$ for pollutant $p$. However, the values are often represented as one of five groups (labels vary by country): good, moderate, unhealthy for sensitive groups, unhealthy, very unhealthy, and hazardous. 
```
Changing the group labels to match those used by our feature

**Review by meretelutz — CHANGES_REQUESTED** [review]
Nice job! Just one small grammar error and changing the labels for consistency with our data.

**Comment by killerninja8** [comment]
Thanks for the suggested changes, @meretelutz! I've committed the changes.

**Pull request merged**